### PR TITLE
Docs: Add a pull request template

### PR DESCRIPTION
**Pull request by tordans**
We can use a pull request template to nudge contributors to add information to the PRs that make it easier to review them.

This template is based on https://github.com/twbs/bootstrap/blob/main/.github/PULL_REQUEST_TEMPLATE.md?plain=1 but change to fit the tagging schema.

My idea is, to nudge contributors to first add more context to PRs like links. But then after the preview was generated, to get back , edit the description and add more links and screenshots that make it super fast to review the PRs.

---

Here are the docs on adding those templates. In general they are much less fancy than the issue templates, so it's just a Markdown template.
https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/creating-a-pull-request-template-for-your-repository

### CODE REVIEW

**Comment by qugebert** [comment]
I like this idea as it helps us newbies to provide all necessary informations.

Maybe we could add a hint in the "Related issues"-Section to use the ```closes #Issue-Number``` Syntax because sometimes issues stay open after the related PR was merged ([Example 1](https://github.com/openstreetmap/id-tagging-schema/issues/148), [Example 2](https://github.com/openstreetmap/id-tagging-schema/issues/25)) and afaik by using ```closes #...``` these Issues are closed automatic after a successful PR.

**Comment by tordans** [comment]
Note to self: New users don't get the preview (and checks) automatically. Once a PR was merged, the checks run for that user automatically.

**Review by tyrasd — DISMISSED** [review]
(testing branch protection configuration)

**Review by tyrasd — DISMISSED** [review]

File: .github/PULL_REQUEST_TEMPLATE.md
Comment (k-yle) [inline .github/PULL_REQUEST_TEMPLATE.md:22]: Alternatively, we could encourage people to come back later and edit the PR description, instead of copy-pasting this template into a separate comment. 

The reason I suggest this is because, it may lead to PRs that contain the empty template in the PR description, and the completed template in a separate comment. 

But this is just a hypothetical problem, if it actually causes confusion we could revisit this in the future

**Review by k-yle — APPROVED** [review]

File: .github/PULL_REQUEST_TEMPLATE.md
Comment (tordans) [reply .github/PULL_REQUEST_TEMPLATE.md:22]: Agreed. I wrote it like this because we will get notifications on new comments … but not on changes to the existing description. But we might need to change that if it does not work…

Comment (tordans) [reply .github/PULL_REQUEST_TEMPLATE.md]: FYI: I reworked this so the initial message is less cluttered.

**Comment by tordans** [comment]
This is a test comment to test the copy-paste template

---

## Test-Documentation

### Preview links & Sidebar Screenshots





### Search



### Info-`i`



### Wording

- [ ] American English
- [ ] `name`, `aliases` (if present) use Title Case
- [ ] `terms` (if present) use lower case, sorted A-Z

**Comment by tordans** [comment]
@tyrasd  this is ready to be merged from my point of view. It will make new PRs a lot easier to review…
I will iterate on it if needed once we learn how people use / understand it.

**Comment by Asteliks** [comment]
Personally, I prefer the description to be in an issue and not duplicated, but I know that this will be annoying for the contributors and maintainers (these groups usually make PRs without issues). As such, my recommendation is that this template should only be used when there isn't an issue linked to the PR

**Comment by tordans** [comment]
> Personally, I prefer the description to be in an issue and not duplicated,

Agreed. However I would assume people are lazy and will just leave this part blank or reference the issue for the first section. 
While reviewing the PRs recently I found that the issue did not really help a lot, which is why I ask for more context here.
The issues are sometimes created by users that only know the wiki link of a tag but don't know how to do more research on it. My goal for the PR is to ask people to do this research now …

I think we can see how it goes with this one and modify if needed.

**Comment by Asteliks** [comment]
> I think we can see how it goes with this one and modify if needed.

I fully agree let's see how it will work in the wiled

**Review by tyrasd — APPROVED** [review]
For a lot of PRs it will not be necessary and perhaps could even be borderline a waste of time to send screenshots, etc.… Hopefully contributors are not deterred by that.

But let's try it anyway and see whether this makes the maintenance work a little bit easier.

**Pull request merged**
